### ISSUE TITLE
Installation errors due to JScript Panel v2.3.0+

### ISSUE THREAD

**Issue by yaerj** [kbuffington/Georgia#20]
```
JScript Panel v2.3.0.1 (Georgia by Mordred)
JavaScript runtime error:
Wrong number of arguments or invalid property assignment
File: C:\Users\Aerjay\Portable\foobar2000\georgia\js\CaTRoX_QWR\Panel_Playlist.js
Line: 2821, Col: 9
```

I got this error after I pasted the contents of `Georgia.txt` to the JScript Panel. It's probably due to the new version of JScript Panel (starting from v2.3.0). In particular there's the following [change](https://github.com/marc2k3/foo_jscript_panel/releases/tag/v2.3.0.1):

` - CHG: plman.PlayingPlaylist is now a read-only property. `

Also, if I play a track before pasting `Georgia.txt` at the panel, this shows up:

```
JScript Panel v2.3.0.1 (Georgia by Mordred)
JavaScript runtime error:
Object doesn't support property or method 'FileTest'
File: C:\Users\Aerjay\Portable\foobar2000\georgia\js\georgia-main.js
Line: 2857, Col: 3
```

this is probably due to this change in JScript Panel:

```
- CHG: utils.FileTest has been removed and replaced with the following 4 methods:
       utils.Chardet(filename)
       utils.GetFileSize(filename)
       utils.IsFile(filename)
       utils.IsFolder(folder)
       There is no replacement for "split" mode.
```

Using JScript Panel v2.2.2.4 instead of the latest one removes the errors.

**Comment by kbuffington** [comment]
Yeah, there's a lot of breaking changes in 2.3.x. I'm currently testing a version with fixes in place and should release it shortly. It's not backwards compatible to earlier versions of jscript, so I need to be extra careful. For now stick with 2.2.2.4

**Comment by ghost** [comment]
I've had a bash at addressing these issues...

https://github.com/kbuffington/Georgia/pull/21

One outstanding issue with Library Tree remains.

**Comment by ghost** [comment]
@yaerj  -~you can help test by downloading this...~

edit: dead link removed. Changes are now part of this repo.

Just don't enable "view by folder structure" in the Library tree - I know that fails.

**Comment by kbuffington** [comment]
Fixed in 1.1.5
